__tests__/wait.test.ts

```diff
@@ -61,6 +62,43 @@ describe("wait", () => {
         ]);
       });
 
+      it("will abort after a prescribed number of seconds", async () => {
+        input.abortAfterSeconds = 1;
+        const inProgressRun = {
+          id: 1,
+          status: "in_progress",
+          html_url: ""
+        };
+        const githubClient = {
+          runs: async (
+            owner: string,
+            repo: string,
+            branch: string | undefined,
+            workflowId: number
+          ) => Promise.resolve([inProgressRun]),
+          workflows: async (owner: string, repo: string) =>
+            Promise.resolve([workflow])
+        };
+
+        const messages: Array<string> = [];
+        const waiter = new Waiter(
+          workflow.id,
+          githubClient,
+          input,
+          (message: string) => {
+            messages.push(message);
+          }
+        );
+        await assert.rejects(
+          waiter.wait(),
+          new Error(`Aborted after waiting 1 seconds`)
```

Comment:
Hi there! I *really* want this feature for Turnstyle, so I thought I'd see if I could fix your testing issue. I got the same error on CI on my local machine, and found a fix.

It looks like the second argument of [`assert.rejects`](https://nodejs.org/api/assert.html#assert_assert_rejects_asyncfn_error_message) should be an object with properties you want to test. So if you change this line to:

```suggestion
          { name: "Error", message: "Aborted after waiting 1 seconds" }
```

then the test passes. Hope this helps ☺️

Reply:
Thanks! I think the docs say it can also be an Error object, but I don't have strong opinions, so if this works, I'm all for it :grin: